tensorflow/python/ops/math_ops.py

```diff
@@ -767,8 +770,7 @@ def sign(x, name=None):
   numpy=array([0.70710678+0.70710678j, 0.        +0.j        ])>
 
   Args:
-   x: A Tensor. Must be one of the following types: bfloat16, half, float32,
-     float64, int32, int64, complex64, complex128.
+   x: A Tensor. Must be one of the following types: bfloat16, half, float, double, int8, int16, int32, int64, complex64, complex128.
```

Comment:
backticks for the various types? e.g. `bfloat16`, `half` ...

Reply:
code: Tensorflow Version（2.11）

    import tensorflow as tf
    input = "ds"
    out = tf.math.sign(input)
    print(out)

result: InvalidArgumentError: Value for attr 'T' of string is not in the list of allowed values: bfloat16, half, float, double, int8, int16, int32, int64, complex64, complex128; NodeDef: {{node Sign}}; Op<name=Sign; signature=x:T -> y:T; attr=T:type,allowed=[DT_BFLOAT16, DT_HALF, DT_FLOAT, DT_DOUBLE, DT_INT8, DT_INT16, DT_INT32, DT_INT64, DT_COMPLEX64, DT_COMPLEX128]> [Op:Sign]

As can be seen from the above code, the type of parameter x supports int8,int16 types.

Reply:
> backticks for the various types? e.g. , ...`bfloat16``half`

I will submit later, and I will also add backticks